src/main/resources/lang/en_US.json

```diff
@@ -222,7 +222,9 @@
     "changeDungeonMapZoomWithKeyboard": "Adjust zoom with +/- keys",
     "showPlayerSymbolsInChat": "Show Player Symbols In Chat",
     "crimsonArmorAbilityStacks": "Show Crimson Armor Stacks",
-    "hideTrueDefense": "Hide True Defense"
+    "hideTrueDefense": "Hide True Defense",
+    "showProfileType": "Show Profile Type",
```

Comment:
Just to clarify, by profile type do you mean "Ironman", "Bingo", etc?

Reply:
Yes, profile type refers to Ironman and bingo(and stranded, but untested).